test/functional/test-async-input.js

```diff
@@ -1,50 +0,0 @@
-/**
- * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS-IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import {AsyncInput} from '../../src/async-input';
-import {createAmpElementForTesting} from '../../src/custom-element';
-
-describes.realWin('BaseElement', {amp: true}, env => {
```

Comment:
test outdated?

Reply:
Yeah it is. 😄  Which is why I removed it? Unless I am misunderstanding?

Reply:
Ah! My bad! Didn't realize this is to remove the test : )